### PR TITLE
Disable failing builds

### PR DESCRIPTION
**Pull request by FredrikAppelros**
* I have checked that this pull request contains only
  `.travis.yml`/`appveyor.yml` changes or commits merged from the `pkg.template` branch. All other changes send
  to https://github.com/ruslo/hunter. **[Yes]**

* I have checked that no toolchains removed from CI configs, they are commented
  out instead so other developers can enable them back easily and to simplify
  merge conflict resolution. **[Yes]**

* I have checked that for every commented out toolchain there is a link to the
  broken CI build page or to the minimum compiler requirements documentation
  so other developers can figure out what was the problem exactly. **[Yes]**

* I have checked that for every enabled toolchain corresponding package passed
  all stages of update cycle: test/merge/upload/release. **[No]**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,23 +74,29 @@ matrix:
...
-        PROJECT_DIR=examples/breakpad
+    # FIXME: OSX client build needs CPlusTest framework
+    # * https://travis-ci.com/FredrikAppelros/hunter/builds/109460348
+    #- os: osx
+    #  osx_image: xcode9.4
+    #  env: >
+    #    TOOLCHAIN=osx-10-13-make-cxx14
+    #    PROJECT_DIR=examples/breakpad
+
+    # FIXME: OSX client build needs CPlusTest framework
```
Comment (ruslo) [inline .travis.yml:85]: I guess you can try to exclude tests?

Comment (FredrikAppelros) [reply .travis.yml:85]: We could. I tried to fix the Android and iOS builds to no avail and eventually just gave up. I have no platform to try this out locally on.

**Comment by FredrikAppelros** [comment]
I don't get the part about:
> I have checked that for every enabled toolchain corresponding package passed all stages of update cycle: test/merge/upload/release

What I have done is tested that the CI tests are still passing:
https://ci.appveyor.com/project/FredrikAppelros/hunter/builds/24088366
https://travis-ci.com/FredrikAppelros/hunter/builds/109562970

**Comment by ruslo** [comment]
> I don't get the part about

tl;dr you don't have **enabled** toolchains, so nothing should be done here.

This one needed if you send a new version to `master`. `pkg.foo` will be updated as the last step of merge/ upload/release cycle.
